backend/src/scripts/example_schemas/minimal_model_schema.json

```diff
@@ -23,6 +23,25 @@
           },
           "uniqueItems": true
         },
+        "metrics": {
+          "type": "array",
+          "title": "Metrics",
+          "widget": "metricsWidget",
+          "items": {
+            "type": "object",
+            "title": "",
+            "properties": {
+              "name": {
+                "title": "Metric name",
+                "type": "string"
+              },
+              "value": {
+                "title": "Model performance metric value",
+                "type": "number"
+              }
+            }
+          }
```

Comment:
Should this have unique items?

Reply:
I don't think that'll work because they're objects